db/migrate/20241017080126_add_last_edited_by_editor_id_to_editions.rb

```diff
@@ -0,0 +1,7 @@
+class AddLastEditedByEditorIdToEditions < ActiveRecord::Migration[7.2]
+  def change
+    change_table :editions, bulk: true do |t|
+      t.uuid :last_edited_by_editor_id, null: true
+    end
```

Comment:
**suggestion (non-blocking):** simplification

This could probably just be an `add_column` given it's a single field